spec/GameSpec.js

```diff
@@ -0,0 +1,216 @@
+describe ('Game', function() {
+
+  beforeEach(function() {
+    game = new Game();
+  });
+
+  it ('starts with 0 scores', function() {
+    expect(game.total).toBe(0);
+  });
+
+  it ('has internally 12 frames', function() {
+    expect(game.frames.length).toBe(12);
+  });
+
+  it ('has 2 rolls in each frame', function() {
+    expect(game.frames[0].length).toBe(2);
+  });
+
+  it ('stores the number of fallen pins in each roll', function() {
+    game.setScores(1, 1, 7);
+    expect(game.frames[0][0]).toBe(7);
+  })
+
+  it ('can check if you spare', function() {
+    game.setScores(1, 1, 8);
+    game.setScores(1, 2, 2);
+    expect(game.isSpare(1)).toBe(true);
+  })
+
+  it ('can check if you spare', function() {
+    game.setScores(1, 1, 8);
+    game.setScores(1, 2, 1);
+    expect(game.isSpare(1)).toBe(false);
+  })
+
+  it ('can check if you strike', function() {
+    game.setScores(1, 1, 10);
+    expect(game.isStrike(1)).toBe(true);
+  })
+
+  it ('counts scores in each frame', function() {
+    game.setScores(1, 1, 7);
```

Comment:
This is repeated quite frequently. Consider having a helper function, something like:

```js
it ('has bonus when you spare', function() {
    setScores([[1, 1, 7],[1, 2, 3], [2, 1, 4]]);
    expect(game.countFrameScores(1)).toBe(14);
  });

  function setScores(scores) {
   // iterate over each score to set it, cleaning up tests
  }
```